libraries/AP_Scripting/docs/docs.lua

```diff
@@ -3177,6 +3177,26 @@ function rangefinder:max_distance_cm_orient(orientation) end
 ---@return integer
 function rangefinder:distance_cm_orient(orientation) end
 
+-- desc
```

Comment:
Full descriptions please.

Reply:
Fixed.  Note that I did just copy the block above which used that "desc" thing :-)

Reply:
> Fixed. Note that I did just copy the block above which used that "desc" thing :-)

Yeah, stuff that pre-dates the docs gets a pass, anything new does not. I need to take another pass at filling in the missing ones.